cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -110,6 +110,7 @@ controller, and serviceaccounts controller.`,
 	return cmd
 }
 
+// get the synchronization Period
```

Comment:
@StudyNick , I think this PR is OK to be merged after addressing comments; and I think @lavalamp 's suggestion is better than mine :).

Anyway, it's up to you to address comments or close it :).